python/mlc_llm/compiler_pass/attach_to_ir_module.py

```diff
@@ -3,7 +3,8 @@
 from typing import Any, Dict
 
 import tvm
-from tvm import IRModule, relax, tir
+from tvm import IRModule, relax, te, tir
+from tvm.relax.frontend import nn
```

Comment:
readability: I think it helps to rename to 

- attach_sampler.py
- attach_logit_processor.py
- attach_embedding_output_allocator.py
- attach_support_info.py

Reply:
Agreed! Now split the file into multiple ones.